tests/e2e/postgresql/00-assert.yaml

```diff
@@ -31,31 +31,6 @@ spec:
     kind: XVSHNPostgreSQL
   writeConnectionSecretToRef:
     name: postgresql-e2e-test-creds
-status:
-  certificateConditions:
-    - status: "True"
-    - status: "True"
-  conditions:
-    - status: "True"
-    - status: "True"
-  localCAConditions:
-    - status: "True"
-    - status: "True"
-  objectBackupConfigConditions:
-    - status: "True"
-    - status: "True"
-  objectBucketConditions:
-    - status: "True"
-    - status: "True"
-  pgclusterConditions:
-    - status: "True"
-    - status: "True"
-  pgconfigConditions:
-    - status: "True"
-    - status: "True"
-  profileConditions:
-    - status: "True"
-    - status: "True"
```

Comment:
why?

Reply:
Because this doesn't work anymore with comp-fucntions and we anyways use events now instead of the status field

Reply:
kk, we need this asap otherwise our chatops will drive me crazy ^^